tests/before_script.sh

```diff
@@ -29,5 +34,5 @@ if [[ $GLPI_BRANCH == "9.2.1" ]] ; then patch -p1 --batch < plugins/flyvemdm/tes
 
 # prepare plugin to test
 cd plugins/flyvemdm 
-composer install --no-interaction
+composer install --no-interaction --ignore-platform-reqs
```

Comment:
I think we should not add this argument, as we discussed. Please,  try to revert this line and let's see how the tests run on  this PR.

Reply:
ok